### ISSUE TITLE
Error. Schematic "1" not found in collection "@nestjs/schematics

### ISSUE THREAD

**Issue by Yerkon** [nestjs/nest-cli#80]
When running `nest generate controller test`. Error occurs:

```
Error: Schematic "1" not found in collection "@nestjs/schematics".
    at SchematicEngine.createSchematic (C:\Users\Erkebulan\AppData\Roaming\npm\node_modules\@nestjs\cli\node_modules\@angular-devkit\schematics\src\engine\engine.js:155:23)
    at CollectionImpl.createSchematic (C:\Users\Erkebulan\AppData\Roaming\npm\node_modules\@nestjs\cli\node_modules\@angular-devkit\schematics\src\engine\collection.js:12:29)
    at NodeWorkflow.execute (C:\Users\Erkebulan\AppData\Roaming\npm\node_modules\@nestjs\cli\node_modules\@angular-devkit\schematics\tools\workflow\node-workflow.js:54:38)
    at Object.<anonymous> (C:\Users\Erkebulan\AppData\Roaming\npm\node_modules\@nestjs\cli\node_modules\@angular-devkit\schematics-cli\bin\schematics.js:172:10)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
Failed to execute command : @nestjs/schematics:1 --name=test --dry-run=false, see above. 
```

nest.js 5.0.0 - Nest CLI

**Comment by ThomRick** [comment]
fixed on version 4.0.1 and 5.0.1

**Comment by leehoisleeho** [comment]
why???
$ nest generate module music           
Error: Path "/Users/leeHo/code/preject_nest/node_modules/@nestjs/schematics/dist/collection.json" does not exist.

Failed to execute command: node @nestjs/schematics:module --name=music --no-dry-run --no-skip-import --language="ts" --source-root="src" --spec --no-flat --spec-file-suffix="spec"
